packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -146,7 +146,7 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   ///
   /// See also:
   ///
-  ///  * [allowedButtonsFilter], which decides which button will be allowed.
+  ///  * `allowedButtonsFilter`, which decides which button will be allowed.
```

Comment:
We'd want all entries of a "See also" section to link to something where you can learn more. I've posted https://github.com/flutter/flutter/pull/150734 to fix that for the entries changed in this PR.